### DISCUSSION TITLE
I didn't receive the 1.75 chia reward

### DISCUSSION THREAD

**Discussion by beyondtwilight** [Chia-Network/chia-blockchain#8848] [Support]
I have soloed for a long time. This morning I found only receive 0.25 XCH for the reward. I didn't receave the 1.75 reward.

![image](https://user-images.githubusercontent.com/92658969/137610163-9a686b27-2aa7-42f9-9149-be700ae7ecbe.png)
![image](https://user-images.githubusercontent.com/92658969/137610135-980fd558-8111-4783-9678-7aee714e893d.png)

I have looked at the transcation the xch has been made to this address
![image](https://user-images.githubusercontent.com/92658969/137610214-0b1d91b4-1ab6-4ad1-acf1-8fcad7403655.png)
It means some has stolen it, but how? Is there any way to ask for return?

**Comment by BrandtH22** [comment]
@beyondtwilight , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
